game/theater/theatergroundobject.py

```diff
@@ -260,17 +263,11 @@ def group_by_name(self, name: str) -> Optional[TheaterGroup]:
                 return group
         return None
 
-    def rotate(self, heading: Heading) -> None:
-        """Rotate the whole TGO clockwise to the new heading"""
-        rotation = heading - self.heading
-        if rotation.degrees < 0:
-            rotation = Heading.from_degrees(rotation.degrees + 360)
-
-        self.heading = heading
-        # Rotate the whole TGO to match the new heading
-        for unit in self.units:
-            unit.position.heading += rotation
-            unit.position.rotate(self.position, rotation)
+    def get_carrier_type(self) -> Type[ShipType]:
```

Comment:
sth went wrong during rebase ;)

Reply:
Indeed! I don't know what happened. This should now be fixed.